### ISSUE TITLE
`isFilled.image` is not fully compatible with `ImageField` type

### ISSUE THREAD

**Issue by sylvaingi** [prismicio/prismic-helpers#41]
### Versions

- @prismicio/helpers: 2.0.0
- node: 16.13.2
- typescript: 4.5.4 (`strict` option enabled)

### Steps to reproduce
When trying to use `isFilled.image` with a variable of type `ImageField` (note that there is no generic type variable, especially for the 1st one, `ThumbnailNames`) tsc complains that the image variable does not have the right type.

```ts
const emptyImage: ImageField = {};
isFilled.image(emptyImage);
// Argument of type 'EmptyImageFieldImage' is not assignable to parameter of type '(EmptyImageFieldImage | FilledImageFieldImage) & Record<string, EmptyImageFieldImage | FilledImageFieldImage>
```

### What is expected?
`isFilled.image` should work with images with no thumbnails, with translates in TS to `ImageField` aka `ImageField<null, FieldState>` since there are default types provided for the type variables.

### What is actually happening?
Code does not typecheck.

**Comment by lihbr** [comment]
Hey @sylvaingi, thank you so much for your contribution! I'll have a look at it and provide a fix. Might only land around the end of the week/beginning of next one though, thanks for your comprehension.

**Comment by sylvaingi** [comment]
No worries, I've sprinkled some `@ts-ignore` comments to fix my build, this is not blocking on my side 👍

**Comment by lihbr** [comment]
Fixed in [2.1.1](https://github.com/prismicio/prismic-helpers/blob/HEAD/CHANGELOG.md#211-2022-02-04) thanks for the catch!
